app/templates/_has_wants_selector.html

```diff
@@ -0,0 +1,182 @@
+                <br/>
+                <table class="table table-striped" id="add-ppe-types">
+                    <thead>
+                        <tr>
+                            <th>Image</th>
+                            <th>SKU</th>
+                            <th>Description</th>
+                            <th>Actions</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for ppe in ppe_types %}
+                        {% if ppe.wants|length > 0 or ppe.has|length > 0 %}
+                            {% set hidden = "hidden" %}
+                        {% else %}
+                            {% set hidden = "" %}
+                        {% endif %}
+                        <tr data-searchable="{{ ppe.desc }} {{ ppe.sku }}" {{ hidden }} id="add-ppe-type-result-{{ ppe.id }}">
+                        {% call ppe_row(ppe) %}
+                            {% if ppe.wants|length > 0 %}
+                                <i>You already want this PPE!</i>
+                            {% elif ppe.has|length > 0 %}
+                                <i>You already have this PPE!</i>
+                            {% else %}
+                                <button class="btn btn-primary add-ppe-type" data-id="{{ ppe.id }}" data-dismiss="modal"><i class="glyphicon glyphicon-plus"></i> Add</button>
+                                <!--
+                                    This is hacky, but it works. We don't want to include templating
+                                    logic on the client, so we render it on the server and store the
+                                    HTML in a script tag with an incorrect type attribute. This
+                                    won't be rendered but will hang out safely till the client-side
+                                    JS reads it.
+                                -->
+                                <script type="text/html" id="ppe-row-template-{{ppe.id}}">
+                                    <tr>
+                                    {{ active_ppe_row(ppe) }}
+                                    </tr>
+                                </script>
+                            {% endif %}
+                        {% endcall %}
+                        </tr>
+                        {% endfor %}
+                    </tbody>
+                </table>
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
+            </div>
+        </div><!-- /.modal-content -->
+    </div><!-- /.modal-dialog -->
+</div><!-- /.modal -->
+
+
+{{ ppe_selector('wants', 'Want') }}
+
+{{ ppe_selector('has', 'Have') }}
+<style>
+    .ppe-container:not(.edit) input:not(.no-edit),
+    .ppe-container:not(.edit) .only-edit,
+    .ppe-container.edit .no-edit {
```

Comment:
Also simple but effective: all the controls needed for editing have a `.only-edit` class, while all the controls that should be hidden during editing have a `.no-edit` class. We use this very simple CSS to hide the other set when needed.